content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -0,0 +1,40 @@
+Testing and running a campaign
+==============================
+
+Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
+them.
+
+First, open the :guilabel:`Marketing Automation` application and click on a campaign. Make sure
+the campaign already has activities configured on it (or build a campaign by following the
+directions here on :doc:`this documentation <workflow_activities>`). To start a test, click the
+:guilabel:`Launch a Test` button at the top of the template form.
+
+.. image:: testing_running/launch-test.png
+   :align: center
+   :alt: Launch a test button in Odoo Marketing Automation.
+
+When clicked, a pop-up window appears. In the dropdown field choose a specific record to run the
+test on, or create a brand new record by clicking the :guilabel:`Search More...` link at the bottom
+of the dropdown menu, and then click the :guilabel:`Create` button.
+
+Once the record is selected, click :guilabel:`Continue`, and Odoo will redirect to the campaign
+test page.
+
+.. image:: testing_running/test-screen.png
```

Comment:
Image is too wide and not in HD.
Replacement [here](https://i.imgur.com/UkpxOVT.png).